pkg/sensors/program/map.go

```diff
@@ -260,12 +261,23 @@ func LoadOrCreatePinnedMap(pinPath string, mapSpec *ebpf.MapSpec) (*ebpf.Map, er
 				"map-name": mapSpec.Name,
 			}).Warn("tetragon, incompatible map found: will delete and recreate")
```

Comment:
Maybe this delete and recreate warn should be moved inside create true?

Reply:
good catch! will move, thanks